backend/tracim_backend/lib/collaborative_document_edition/collaboration_document_edition_factory.py

```diff
@@ -0,0 +1,52 @@
+import typing
+
+from sqlalchemy.orm import Session
+
+from tracim_backend.config import CFG
+from tracim_backend.exceptions import NoValidCollaborativeDocumentEditionSoftware
+from tracim_backend.lib.collaborative_document_edition.data import COLLABORA_DOCUMENT_EDITION_SLUG
+from tracim_backend.models.auth import User
+
+
+class CollaborativeDocumentEditionFactory(object):
+    """
+    Factory to get correct objects related to search engine activated
+    """
+
+    @classmethod
+    def get_collaborative_document_edition_controller(cls, config: "CFG"):
```

Comment:
- missing typing
- `get_controller` would be shorter and as explicit as `get_collaborative_document_edition_controller`

Reply:
done